network_wrangler/TransitNetwork.py

```diff
@@ -47,35 +48,24 @@ def read(self, agency_file: str, frequencies_file: str, routes_file: str,
         transfers_file: the transfers file
         trips_file: the trips file
         path: the location of all the network standard files
-        '''
-        
-        
-        #TODO: not sure which column I should be passing for the second dictionary.
-        #I thought it might be the ID column but their examples are not consistent with that
-        #and their documentation is horrible.... Any suggestions?
-        view = {agency_file: {'service_id': None},
-                frequencies_file: {'service_id': None},
-                routes_file: {'service_id': None},
-                shapes_file: {'service_id': None},
-                stops_files: {'service_id': None},
-                transfers_file: {'service_id': None},
-                trips_file: {'service_id': None},
-                }
-        
+        '''     
         
         if not os.path.exists(path):
             WranglerLogger.debug("\nPath [%s] doesn't exist; creating." % path)
             os.mkdir(path)
             
-        #TODO: do we want load_feed or load_geo_feed?
-        feed = ptg.load_feed(path, view)
+        config = geo_config()
+        #using load_feed so we can feed a config to it
+        feed = ptg.load_feed(path, view = None, config = config)
+        
         
+        #TODO: the above seems to be reading trips, routes, frequencies and agencies. Shapes, stops and transfers are not read.
```

Comment:
its failing on shapes b/c the headers don't match what it is looking for in: https://github.com/remix/partridge/blob/0ba80fa30035e5e09fd8d7a7bdf1f28b93d53d03/partridge/config.py#L251

Reply:
do we have to fix our input file? Who does that? Sijia?